docs/faq.md

```diff
@@ -2,119 +2,160 @@
 
 ## What browsers will Gutenberg support?
 
 Gutenberg will work in modern browsers, and Internet Explorer 11.
 
 ## How do I make my own block?
-The API for creating blocks is a crucial aspect of the project. We are working on improved documentation and tutorials. Current documentation lives here: https://github.com/WordPress/gutenberg/tree/master/blocks
 
-## Does Gutenberg involve editing posts/pages in the frontend?
-No, Gutenberg is a replacement for the post and page editing screens.
+The API for creating blocks is a crucial aspect of the project. We are working on improved documentation and tutorials. Current documentation lives here
+
+## Does Gutenberg involve editing posts/pages in the front-end?
+
+No, we are designing Gutenberg primarily as a replacement for the post and page editing screens. That said, front-end editing is often confused with an editor that looks exactly like the front end. And that is something that Gutenberg will allow as themes customize individual blocks and provide those styles to the editor. Since content is designed to be distributed across so many different experiences—from desktop and mobile to full-text feeds and syndicated article platforms—we believe it's not ideal to create or design posts from just one front-end experience.
 
 ## Given Gutenberg is built in JavaScript, how will old metaboxes (PHP) work?
-We expect to have a dedicated "advanced" area at the bottom of the editor. It’s not yet implemented, we are tracking it here: https://github.com/WordPress/gutenberg/issues/952
+
+We expect to have a dedicated "advanced" area at the bottom of the editor. It’s not yet implemented, and we are tracking development here: https://github.com/WordPress/gutenberg/issues/952
 
 ## How can plugins extend the Gutenberg UI?
-The main extension point we want to emphasize is creating new blocks. We are still working on how to extend the rest of the UI that is build in JS. We are tracking it here: https://github.com/WordPress/gutenberg/issues/1352
+
+The main extension point we want to emphasize is creating new blocks. We are still working on how to extend the rest of the UI that is built in JS. We are tracking it here: https://github.com/WordPress/gutenberg/issues/1352
 
 ## Will Custom Post Types be supported?
-Yes, for the most part. If a post type disables the content field the idea is the "advanced" section at the bottom would fill the page.
+
+Yes, for the most part. If a post type disables the content field, the "advanced" section at the bottom would fill the page.
 
 ## Will there be columns?
-Columns are not meant to be part of the initial version since we want to focus on a solid block foundation first. As Gutenberg ties into the following customization focus, things like columns are going to be explored.
+
+Our primary goal is on a solid block foundation before exploring column support.
```

Comment:
Worth a reference to nested blocks, at least in how it would enable a path forward for columns? (Even though it is the very next question)